src/modules/token/fungible/hooks/queries/useTokensBalance.js

```diff
@@ -20,17 +18,20 @@ import { useCustomInfiniteQuery } from 'src/modules/common/hooks';
  * @returns the query object
  */
 // eslint-disable-next-line import/prefer-default-export
-export const useTokensBalance = ({ config: customConfig = {}, options } = {}) => {
+export const useTokensBalance = ({ config: customConfig = {}, options, client } = {}) => {
+  const [currentAccount] = useCurrentAccount();
+  const { address } = currentAccount.metadata;
```

Comment:
How would it work in scenario where "WalletConnect" Requests for signature, are we setting the active account before hand?

Reply:
I think the account has to be selected before hand or possibly if there are not current or active accounts, on request to sign a transaction by WaletConnect, the user can select an active account at that moment. @ManuGowda 

@soroushm more clarification please 🙏🏼